test/OpenTelemetry.Exporter.Prometheus.Tests/PrometheusExporterMiddlewareTests.cs

```diff
@@ -36,12 +38,156 @@ public sealed class PrometheusExporterMiddlewareTests
         private static readonly string MeterName = Utils.GetCurrentMethodName();
 
         [Fact]
-        public async Task PrometheusExporterMiddlewareIntegration()
+        public Task PrometheusExporterMiddlewareIntegration()
         {
-            var host = await new HostBuilder()
+            return RunPrometheusExporterMiddlewareIntegrationTest(
+                "/metrics",
+                app => app.UseOpenTelemetryPrometheusScrapingEndpoint());
+        }
+
+        [Fact]
+        public Task PrometheusExporterMiddlewareIntegration_Options()
+        {
+            return RunPrometheusExporterMiddlewareIntegrationTest(
+                "/metrics_options",
+                app => app.UseOpenTelemetryPrometheusScrapingEndpoint(),
+                services => services.Configure<PrometheusExporterOptions>(o => o.ScrapeEndpointPath = "metrics_options"));
+        }
+
+        [Fact]
+        public Task PrometheusExporterMiddlewareIntegration_OptionsFallback()
+        {
+            return RunPrometheusExporterMiddlewareIntegrationTest(
+                "/metrics",
+                app => app.UseOpenTelemetryPrometheusScrapingEndpoint(),
+                services => services.Configure<PrometheusExporterOptions>(o => o.ScrapeEndpointPath = null));
```

Comment:
Seems unlikely that someone would set this to null and expect the path to result in `/metrics`. Should we fail initialization in this case?

Reply:
Oh, is this the case where options are read from configuration - like a file, environment, etc? If so, does null in this context effectively mean the user didn't specify it and that the default would be expected?

Reply:
@alanwest What this test is verifying is that if the user does mess up and specify `null` (either from code or through config binding) for the endpoint path we fallback to the default path. I could switch it to a throw (fast-fail) on startup for that case. Might be a better experience for users that way 🤔